aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -9,6 +9,7 @@ proposals:
     execution_mode: MultiStep
     update_sequence:
       - DefaultGasWithOverrideOld:
+          feature_version: 15
```

Comment:
since 15 is enabled on mainnet already. shouldn't it be the default? do we still need to specifically call it out?

Reply:
Without it, it assumes `LATEST_GAS_FEATURE_VERSION` which is 16 in the v1.11 branch. Step 1 would accidentally upgrade the gas feature version to 16 and do the override. This is not what we want.

We want the gas feature version to remain 15 during Step 1 and be upgraded to 16 in Step 2. (cc: @vgao1996 )